packages/configs/eslint/configs/index.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  env: { node: true },
+  plugins: ['@typescript-eslint'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+
+    // always extend last to override previous extensions
+    'prettier',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  rules: {
+    '@typescript-eslint/explicit-module-boundary-types': 2,
+    '@typescript-eslint/member-ordering': 'error',
+    '@typescript-eslint/no-extra-semi': 'error',
+    '@typescript-eslint/no-floating-promises': 'off',
+    '@typescript-eslint/no-unused-expressions': [
+      'error',
+      { allowTernary: true },
+    ],
+    '@typescript-eslint/no-use-before-define': 'error',
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      { argsIgnorePattern: '_', varsIgnorePattern: '_' },
+    ],
+    '@typescript-eslint/prefer-nullish-coalescing': 'error',
+    '@typescript-eslint/restrict-template-expressions': 'off',
+    '@typescript-eslint/unbound-method': 'off',
```

Comment:
Turning this off since it is doesn't provide much benefit for us (the rule is related to `class` variables). More info [here](https://typescript-eslint.io/rules/unbound-method/)